tests/unit_test/app/routes/test_git_tokens.py

```
@@ -0,0 +1,259 @@
+from unittest.mock import AsyncMock, MagicMock, patch
+from urllib.parse import quote
+
+from fastapi import status
+
+from app.routes.git_tokens import mask_token
+
+
+class TestMaskToken:
+	"""Test cases for token masking functionality."""
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)